generate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
```

Comment:
For portability, I think `#!/usr/bin/env sh` or `#!/usr/bin/env bash` is better

Reply:
yeah sorry, I was just trying to make it work quickly on my machine. I will swap in one of those